generic2d/build.gradle

```
@@ -27,4 +27,6 @@ dependencies {
     implementation project(':tweetwallfx-2d')
     implementation project(':tweetwallfx-core')
     implementation project(':tweetwallfx-tweet-api')
+    implementation project(':tweetwallfx-stepengine-steps')
+    implementation project(':tweetwallfx-tweet-impl-twitter4j')
```

An API implementation as implementation dependency?

```suggestion
    runtimeOnly project(':tweetwallfx-tweet-impl-twitter4j')
```